conjure-java-core/src/test/resources/ete-service.yml

```diff
@@ -132,3 +135,36 @@ services:
 
       noReturn:
         http: POST /no-return
+
+      enumQuery:
+        http: GET /enum/query
+        args:
+          queryParamName:
+            param-type: query
+            type: SimpleEnum
+        returns: SimpleEnum
+
+      enumListQuery:
+        http: GET /enum/list/query
+        args:
+          queryParamName:
+            param-type: query
+            type: list<SimpleEnum>
```

Comment:
interestingly, the wire format spec doesn't allow this: `If the param-type is path, header or query then the de-aliased type MUST be an enum, a primitive or an optional of one of these. `

https://palantir.github.io/conjure/#/docs/spec/conjure_definitions

Reply:
Opened palantir/conjure#191